modules/acquia_telemetry/acquia_telemetry.info.yml

```diff
@@ -2,6 +2,14 @@ name: 'Acquia Telemetry'
 core: 8.x
 type: module
 package: Lightning
-description: 'Sends anonymized data about Drupal and Acquia product usage for product development purposes.'
+description: "By installing the Acquia Telemetry module, you agree to allow it to send anonymized telemetry data to Acquia for the purposes and on the conditions set forth below. If you do not agree, do not install this software.
+
+The Acquia Telemetry module sends anonymized data about Acquia product usage to Acquia for security, operations management, statistical analyses, and for research and development purposes. Specifically, it sends the following data:<br />
```

Comment:
Should we maybe include some P tags around the non-list parts of this, to make it a consistent block of HTML? Additionally, could you post a screenshot of how this renders?